<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plot</class>
 <widget class="QMainWindow" name="plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>plot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="panel">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolButton" name="btn_open">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_opengl">
          <property name="text">
           <string>OpenGL</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_group">
          <property name="text">
           <string>Group</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_step"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="graph_t" name="graph" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>graph_t</class>
   <extends>QWidget</extends>
   <header>graph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
